bundles/org.openhab.binding.doorbird/README.md

```diff
@@ -0,0 +1,217 @@
+# Doorbird Binding
+
+Binding for Doorbird D101 and D210x video doorbells.
+
+## Supported Things
+
+Two thing types are supported:
+
+- Doorbird D101
+- Doorbird D210x
+
+## Discovery
+
+Auto-discovery is not supported at this time.
+
+## Thing Configuration
+
+The following configuration parameters are available on the Doorbird thing:
+
+| Parameter | Required/Optional | Description |
+|-----------|-------------------|-------------|
+| ID | Required | The ID number of the Doorbird device. This is usually a single digit (e.g. 1). |
+| Hostname | Required | The ID number of the Doorbird device. This is usually a single digit (e.g. 1). |
+| User ID | Required | User Id of a Doorbird user that has permissions to access the camera, motion sensor, etc. The User ID and Password must be created using the Doorbird smart phone application.
+| Password | Required | Password of a Doorbird user. |
+| Image Refresh Rate | Optional | Rate at which image channel should be automatically updated. Leave field blank (default) to disable refresh. |
+| Doorbell Off Delay | Optional | Number of seconds to wait before setting doorbell channel OFF after a doorbell event. Leave field blank to disable. |
+| Motion Off Delay | Optional | Number of seconds to wait before setting motion channel OFF after a motion event. Leave field blank to disable. |
+| Montage Number of Images | Required | Number of images to include in the doorbell and motion montage images. Default is 0. |
+| Montage Scale Factor | Required | Percent scaling factor for montage image. Default is 100. |
+
+
+## Channels
+
+The following channels are supported by the binding.
+
+| Channel ID | Item Type | Description |
+|------------|-----------|-------------|
+| doorbell | Switch | Updated to ON whenever doorbell is pressed |
```

Comment:
I think this could be a trigger channel (not sure if they are still available in openHAB, they were year ago).
This would simplify the code, no need for delayed setting state to Off. Same for motion trigger.

Reply:
This is a good idea. As there are a number of people using the binding, I'll post this for discussion in the forum to get some feedback.

Reply:
This is a good idea. Since there are a number of people using the binding already, to ensure backwards compatibility, I could make this a trigger channel, then use a profile to retain the ability to link a Switch item to the channel. WDYT?

Reply:
I just learned and read about profiles and it looks like a great idea

Reply:
Ok, I'll give that a go. I've not used profiles before, so this will be a good opportunity to learn how to use them.

Reply:
I've implemented the trigger and profile on the doorbell and motion channels. I'm pretty happy with how it's working, so I'm going to push the change. I'm also planning to leave the jobs that set the state to OFF. At least for the `motion` channel, that's one of the few ways one can mitigate the Doorbird's hyperactive motion sensor...